packages/manager/modules/pci/src/components/project/flavors-list/flavors-list.component.js

```diff
@@ -19,5 +19,8 @@ export default {
     serviceName: '@',
     singleFlavorWarning: '@?',
     flavorCount: '<?',
+    reload: '<',
+    loadEnd: '&',
+    currentStep: '<?',
```

Comment:
**nitpick:** Since `flavorList` is a generic independent component, instead of accepting `currentStep` can you accept the input as `displayLoader`. This way if we want to reuse the component in other places it is easy to handle the behavior.

Reply:
Done